manifest/ManifestConfigError.ts

```
@@ -0,0 +1,10 @@
+//this error says that there is something wrong with node configuration
+//(ie. manifest file) and we should stop immediately.
+//note that error message should give user a hint what exactly is missing/wrong
+//with the configuration
+export default class ManifestConfigError extends Error {
```

Not a big fan of this solution, but I don't have a better idea how to differ between "configuration errors" (which IMO should fail fast and stop the node) and "processing/api errors" (which in general should be only logged)

---

alternatively I was thinking about some kind of Manifest Validator, that would be fired on Node creation.